### PR TITLE
made flag styling consistent

### PR DESCRIPTION
**Pull request by devidask27**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -443,7 +443,6 @@ Usage:
 Flags:
   -h, --help   help for deauthorize-node
       --mfa string   MFA token
-  -s, --skipCheck    skips node check
```
Comment (nikhilbandi) [inline README.md:446]: Don't we have to replace this with --skip-check?

Comment (devidask27) [reply README.md:446]: This flag does not exist currently in deauthorize-node command. so removed it

**Review by nikhilbandi — CHANGES_REQUESTED** [review]
Added 1 minor comment.

File: README.md (hunk 1 above)
Comment (nikhilbandi) [reply README.md:446]: OK.

**Review by nikhilbandi — APPROVED** [review]
Approving.

**Pull request merged**